frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -105,81 +113,77 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
             path: '/monitoring/alertmanagerconfig',
             menuItems: [
               {
-                label: 'Create Receiver',
+                label: t('cluster-settings~Create Receiver'),
                 href: '/monitoring/alertmanagerconfig/receivers/~new',
               },
               {
-                label: `Edit Configuration YAML`,
+                label: t('cluster-settings~Edit configuration YAML'),
                 href: `/monitoring/alertmanageryaml`,
               },
             ],
           },
         ]);
       const sortedItems = _.sortBy(_.flatten(allItems), 'label', 'asc');
-
-      this.setState({
-        items: sortedItems,
-        loading: false,
-      });
-    });
-  }
-
-  render() {
-    const { errors, items = [], loading, textFilter } = this.state;
-    const visibleItems = items.filter(({ label, description = '' }) => {
-      return (
-        fuzzyCaseInsensitive(textFilter, label) ||
-        description.toLowerCase().indexOf(textFilter.toLowerCase()) !== -1
-      );
+      setItems(sortedItems);
+      setLoading(false);
     });
+  }, [configResources, errors, globalConfigs, oauthMenuItems, t]);
+  const visibleItems = items.filter(({ label, description = '' }) => {
     return (
-      <>
-        <div className="co-m-pane__body">
-          {!loading && (
-            <>
-              <p className="co-help-text">
-                Edit the following resources to manage the configuration of your cluster.
-              </p>
-              <div className="co-m-pane__filter-row">
-                <TextFilter
-                  value={textFilter}
-                  label="by name or description"
-                  onChange={(val) => this.setState({ textFilter: val })}
-                />
-              </div>
-            </>
-          )}
-          {!_.isEmpty(errors) && (
-            <ExpandableAlert
-              variant={AlertVariant.danger}
-              alerts={errors.map((error, i) => (
-                <div key={i}>{error}</div>
-              ))}
+      fuzzyCaseInsensitive(textFilter, label) ||
+      description.toLowerCase().indexOf(textFilter.toLowerCase()) !== -1
+    );
+  });
+
+  return (
+    <div className="co-m-pane__body">
+      {!loading && (
+        <>
+          <p className="co-help-text">
+            {t(
+              'cluster-settings~Edit the following resources to manage the configuration of your cluster.',
+            )}
+          </p>
+          <div className="co-m-pane__filter-row">
+            <TextFilter
+              value={textFilter}
+              label={t('cluster-settings~by name or description')}
```

Comment:
We probably need to change how this works in the component because we can't concat `Filter {label}...` as we were doing before. It needs to be one string for translation. That should be done in the common components story, though.

cc @rebeccaalpert